docs/Installation.md

```diff
@@ -41,34 +40,55 @@ minutes to complete.
 Building and flashing the micro-controller
 ==========================================
 
+Via Octoprint, make sure that there is no active serial connection to the printer.
+Fom the OctoPrint web page, under the "Connection" section, click "Disconnect"
+
 To compile the micro-controller code, start by running these commands
-on the Raspberry Pi:
+on the Raspberry Pi via an SSH command line session:
 
 ```
 cd ~/klipper/
 make menuconfig
 ```
 
-Select the appropriate micro-controller and review any other options
-provided. Once configured, run:
+Manually select the appropriate micro-controller, processor speed (mhz), and baurdrate. 
+Review any other options provided. After choosing the correct settings for your printers
+micro-controller, you will exit the configuration window by choosing 'Exit'. Then select
+'Yes' to save your new configuration.  
+
+Compile the Klipper firmware by entering:
 
 ```
 make
 ```
 
-Finally, for common micro-controllers, the code can be flashed with:
+Ensure that the Klipper service is not currently by entering:
 
 ```
 sudo service klipper stop
-make flash FLASH_DEVICE=/dev/ttyACM0
-sudo service klipper start
+``` 
+
+Identify the communications port that will be used to upload the file to your microncontroller
+using the following command. 
+
+```
+ls -l /dev/serial/by-id/
 ```
 
-When flashing for the first time, make sure that OctoPrint is not
-connected directly to the printer (from the OctoPrint web page, under
-the "Connection" section, click "Disconnect"). The most common
-communication device is **/dev/ttyACM0** - see the
-[FAQ](FAQ.md#wheres-my-serial-port) for other possibilities.
+The most common communication device ports are **/dev/ttyUSB0** or **/dev/ttyACM0**
+Please see the [FAQ](FAQ.md#wheres-my-serial-port) if you are unable to identify the specific port exists.
+
+Next, use the serial port info gathered to flash the klipper firmware to your device using the following command:
+
+```
+make flash FLASH_DEVICE=/dev/ttyACM0 or, make flash FLASH_DEVICE=/dev/ttyUSB0
```

Comment:
you wrote "using the following command" followed by 2 separate commands in one line and "or" in between, but still in the command style, this is rather confusing.

Reply:
It's only confusing if you didn't identify which port you should be using to begin with. There should be an example here even if there is only one command used.

Reply:
it's confusing because you are saying to run a command that isn't actually a functioning command. Using command markdown for an "or" that isn't part of the command is not something that you want in an instruction document.

Reply:
You are arguing about the word OR.

Remove the OR. Use one or the other but recognize both examples need to be there.

Reply:
I'm arguing about the way you use it and make the instructions much more confusing than they previously. I also don't agree that both examples **need** to be there, as there are many more paths that can go there, not just the 2 you happened to find. But if you want to have both, that's fine, just make it clear that they **are** in fact 2 separate command, not one long single command, because that's making it worse.

Reply:
> Thanks. Unfortunately, this change request fails the automatic build tests. Given the feedback in issue #1257, I've updated "make menuconfig" to not present the load/save buttons. I'd like to keep the main documentation as is for now and gather additional feedback. If I see a further trend of users being confused by the build then I will look to make a change at that time.
> 
> -Kevin

Could you give me an idea of why it failed when I only changed text?